.gitlab/ci/on-push.yml

```diff
@@ -76,15 +93,14 @@ create-instances:
         ./Tests/scripts/upload_artifacts.sh
       fi
 
-
 .test_content_on_server_instances_base:
   extends:
     - .default-job-settings
     - .push-rule
   needs: ["create-instances"]
   stage: run-instances
-  dependencies:
-    - create-instances
+  variables:
+    DESTROY_INSTANCES: "true"
```

Comment:
Where is this environment variable used?

Reply:
in the after_script